src/emoji/EmojiModel.cpp

```diff
@@ -58,10 +59,12 @@ EmojiModel::data(const QModelIndex &index, int role) const
             return Provider::emoji[index.row()].unicode;
 
         case Qt::ToolTipRole:
```

Comment:
Can you make the tooltip include both shortname and unicode, if they are different and not empty?

Reply:
```cpp
case Qt::ToolTipRole:
{
    if (Provider::emoji[index.row()].shortcode.isEmpty())
        return Provider::emoji[index.row()].unicode;
    else
        return Provider::emoji[index.row()].shortcode + ", " + Provider::emoji[index.row()].unicode:
}
```